src/atoms/Input/Input.story.tsx

```diff
@@ -14,7 +14,7 @@ import Panel from '../Panel';
 import Input, { InputProps } from './Input';
 
 class ControlledInput extends Component<
-  InputProps &
+  InputProps<any> &
```

Comment:
Is it possible to make this more strict than `any` (here and in other places)

Reply:
Maybe! I'm not the most proficient in TypeScript yet, so if anyone has suggestions, feel free!